test/common/firefox-cdp-driver.js

```diff
@@ -47,13 +47,29 @@ function fatal() {
     process.exit(1);
 }
 
-function fail(err) {
+// We keep sequence numbers so that we never get the protocol out of
+// synch with re-ordered or duplicate replies.  This only matters for
+// duplicate replies due to destroyed contexts, but that is already so
+// hairy that this big hammer seems necessary.
+
+var cur_cmd_seq = 0;
+var next_reply_seq = 1;
```

Comment:
Could these both start at 0 or 1, and both use the same increment? Right now, `cur_cmd_seq` uses pre-increment and `next_reply_seq` post-increment; but they both track the same thing, so let's please make this consistent.

Reply:
They could, but the executionContextDestroyed would have code like `fail(next_cmd_seq-1, ...)` which is also awkward.  I have changed the code a bit to always use `... += 1` for the counters.

Reply:
You didn't actually [change the code](https://github.com/cockpit-project/cockpit/compare/edec865986b2e0e37f7d34be77965185da16596b..1b521e01ba7046424ebe3e6d2db739c76f3437a2), but ok; agreed we don't want the -1 either. That's not what I had in mind, but bikeshedding.